pytest_dotenv/plugin.py

```diff
@@ -1,18 +1,21 @@
 # -*- coding: utf-8 -*-
 
-from dotenv import load_dotenv
-
 import pytest
+from dotenv import load_dotenv
 
 
 def pytest_addoption(parser):
     parser.addini("env_files",
                   type="linelist",
                   help="a line separated list of env files to parse",
                   default=[])
+    parser.addini("env_files_override",
+                  type="bool",
+                  help="override the existing environment variables",
+                  default=[])
 
 
 @pytest.hookimpl(tryfirst=True)
 def pytest_load_initial_conftests(args, early_config, parser):
     for file in early_config.getini("env_files"):
-        load_dotenv(file)
+        load_dotenv(file, override=early_config.getini("env_files_override"))
```

Comment:
Could you also adapt the README and provide examples how the ini file should look like for the user?